File: docs/datatools/codesnippet/CSharp/edit-data-in-datasets_3.cs
            NorthwindDataSet.CustomersRow newCustomersRow = 
                northwindDataSet1.Customers.NewCustomersRow();

            newCustomersRow.CustomerID = "ALFKI";
            newCustomersRow.CompanyName = "Alfreds Futterkiste";

            northwindDataSet1.Customers.Rows.Add(newCustomersRow);

File: docs/data-tools/codesnippet/VisualBasic/edit-data-in-datasets_3.vb
        Dim newCustomersRow As NorthwindDataSet.CustomersRow
        newCustomersRow = NorthwindDataSet1.Customers.NewCustomersRow()

        newCustomersRow.CustomerID = "ALFKI"
        newCustomersRow.CompanyName = "Alfreds Futterkiste"

        NorthwindDataSet1.Customers.Rows.Add(newCustomersRow)